### PR TITLE
implemented generic GET request.

### PR DESCRIPTION
**Pull request by mdameen21122016**
implemented generic GET request, please review and merge.

### CODE REVIEW

File: app/routers/product.py [hunk 1]
```diff
@@ -24,6 +24,12 @@ def add_new_product(payload: schema.Product, db: Session = Depends(get_db)):
     return {"data": new_product}
 
 
+@router.get("/")
```
Comment (mdmubeenkhan) [inline app/routers/product.py:27]: Can you explain this method

File: app/test.py
Comment (mdmubeenkhan) [inline app/test.py:1]: Why this file is added?

**Review by mdmubeenkhan — COMMENTED** [review]
Please resolve the review comments.

File: app/test.py
Comment (mdameen21122016) [reply app/test.py:1]: This file is added to implement tests.

File: app/routers/product.py (hunk 1 above)
Comment (mdameen21122016) [reply app/routers/product.py:27]: this method provide information when requested using GET.

**Pull request merged**